packages/admin/admin-generator/src/commands/generate/generate-command.ts

```diff
@@ -184,7 +185,26 @@ export type GridConfig<T extends { __typename?: string }> = {
     selectionProps?: "multiSelect" | "singleSelect";
 };
 
-export type GeneratorConfig<T extends { __typename?: string }> = FormConfig<T> | GridConfig<T> | TabsConfig<T>;
+type CrudPageGridConfig = {
+    component: ComponentType;
+};
+
+export type CrudPageFormConfig = {
+    component: ComponentType;
+    pageTitle?: string;
+};
+
+export type CrudPageConfig<T extends { __typename?: string }> = {
+    type: "crudPage";
+    topLevelTitle?: string;
+    gqlType: T["__typename"];
+    grid: CrudPageGridConfig;
```

Comment:
wouldn't `grid: ComponentType` be preferrable? you could also use `ComponentType | { component: ComponentType; pageTitle?: string }`

Reply:
Good point, I changed it to `ComponentType`, we can add the `| { ... }` when we add additional settings in the future.